sdk/framework/openapi.go

```diff
@@ -524,7 +524,11 @@ func documentPath(p *Path, specialPaths *logical.Paths, requestResponsePrefix st
 			}
 		}
 
-		doc.Paths["/"+path] = &pi
+		openAPIPath := "/" + path
+		if doc.Paths[openAPIPath] != nil {
```

Comment:
```suggestion
		if _, ok := doc.Paths[openAPIPath]; ok {
```

This might be a more idiomatic check

Reply:
I started out writing it that way, but then I changed it to be consistent in style with these other usages in the same file :-)

https://github.com/hashicorp/vault/blob/e32cf520f43c2708a7f6833f86d4fc0bd5620b00/sdk/framework/openapi.go#L322
https://github.com/hashicorp/vault/blob/e32cf520f43c2708a7f6833f86d4fc0bd5620b00/sdk/framework/openapi.go#L328
https://github.com/hashicorp/vault/blob/e32cf520f43c2708a7f6833f86d4fc0bd5620b00/sdk/framework/openapi.go#L421